web/frontend/src/components/camera/camera.vue

```diff
@@ -43,38 +44,23 @@ const manageStreamStates = (cameraIsOn: boolean) => {
   camerasOn = counter;
 };
 
-const viewCamera = async (isOn: boolean) => {
+const viewCamera = async () => {
   const streams = new StreamClient(props.client);
-
-  streams.on('track', (event) => {
-    let [eventStream] = event.streams;
-    eventStream = event.streams[0];
-    if (!eventStream) {
-      throw new Error('expected event stream to exist');
-    }
-    // Ignore event if received for the wrong stream, in the case of multiple cameras
-    if (eventStream.id !== props.cameraName) {
-      return;
-    }
-    videoStream = eventStream;
-  });
-
-  if (props.refreshRate === 'Live') {
-    manageStreamStates(isOn);
-
-    if (camerasOn === 1) {
-      try {
-        await streams.add(props.cameraName);
-      } catch (error) {
-        displayError(error as ServiceError);
-      }
-    } else if (camerasOn === 0) {
-      try {
-        await streams.remove(props.cameraName);
-      } catch (error) {
-        displayError(error as ServiceError);
+  if (camerasOn === 1 && props.toggle) {
+    try {
+      await streams.add(props.cameraName);
+    } catch (error) {
+      const tempError = error as ServiceError;
+      if (tempError.message !== 'stream already active') {
+        displayError(tempError as ServiceError);
       }
```

Comment:
Though I don't like it, we are going to fix with a stream manager refactor soon. So I'm accepting this for now. It doesn't affect streams working correctly.